tests/test_dashboard.py

```diff
@@ -45,5 +47,78 @@ def test_modal_buttons(self):
         # assert create_project_modal.institutions_are_selected(institutions)
 
         create_project_modal.cancel_button.click()
-        with pytest.raises(ValueError):
-            create_project_modal.modal
+        create_project_modal.modal
+
+class TestDashboardPageProjectList(SeleniumTest):
+
+    def setup_method(self, method):
+        self.dashboard_page = DashboardPage(self.driver)
+        self.dashboard_page.goto()
+
+    @pytest.fixture()
+    def project_one(self):
+        project_one = osf.create_project(self.session, title='&&aaaaaa')
+        yield project_one
+        project_one.delete()
+
+    @pytest.fixture()
+    def project_two(self):
+        project_two = osf.create_project(self.session, title='&&aaaabb')
+        yield project_two
+        project_two.delete()
```

Comment:
Ooh, that's super handy. I didn't know about the yield-delete pattern with fixtures.

Reply:
it's one of my most favorite things to do in pytest. Very convenient. We need to find a way to delete projects created with UI when testing project creation. Maybe run a script daily to clean the test user up.